aider/commands.py

```diff
@@ -495,6 +495,34 @@ def cmd_run(self, args, add_on_nonzero_exit=False):
             )
             return msg
 
+    # Add cmd_rg for calling ripgrep with -l and the user-provided arguments. Then call cmd_add with the returned file names.
+    def cmd_rg(self, args):
```

Comment:
Does this library have unit/integration tests for commands? If not, I would paste a screenshot of this working locally.